sdk/javascript/README.md

```diff
@@ -0,0 +1,92 @@
+
+[sdk-javascript-job]: https://jenkins.symboldev.com/blue/organizations/jenkins/Symbol%2Fgenerated%2Fsymbol%2Fjavascript/activity?branch=dev
+[sdk-javascript-lint]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fsymbol%2Fjavascript%2Fdev%2F&config=sdk-javascript-lint
+[sdk-javascript-test]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fsymbol%2Fjavascript%2Fdev%2F&config=sdk-javascript-test
+[sdk-javascript-vectors]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fsymbol%2Fjavascript%2Fdev%2F&config=sdk-javascript-vectors
+[sdk-javascript-cov]: https://codecov.io/gh/symbol/symbol/branch/dev/graph/badge.svg?token=SSYYBMK0M7&flag=sdk-javascript
+[sdk-javascript-cov-link]: https://codecov.io/gh/symbol/symbol/tree/dev/sdk/javascript
+[sdk-javascript-package]: https://img.shields.io/npm/v/symbol-sdk-javascript
+[sdk-javascript-package-link]: https://www.npmjs.com/package/symbol-sdk-javascript
+
+
+This is new, minimal Javascript SDK for Symbol and NEM.
+The SDK will share similarities with SDKs for other languages.
+Old typescript SDK has been deprecated.
+
+Note, at this point NEM support should be considered **experimental**.
+
+## Changes between 2.0.0 and 3.x
+
+### SDK
+
+NEM and Symbol features are grouped under facades.
+
+1. Transactions can be created using so called 'descriptors'.
+
+Symbol:
+
+```javascript
+const { SymbolFacade } = require('symbolSdk/src/facade/SymbolFacade');
+...
+const facade = new SymbolFacade('testnet');
+const transaction = facade.transactionFactory.create({
+	type: 'transfer_transaction',
+	signerPublicKey: '87DA603E7BE5656C45692D5FC7F6D0EF8F24BB7A5C10ED5FDA8C5CFBC49FCBC8',
+	fee: 1000000n,
+	deadline: 41998024783n,
+	recipientAddress: 'TCHBDENCLKEBILBPWP3JPB2XNY64OE7PYHHE32I',
+	mosaics: [
+		{ mosaicId: 0x7CDF3B117A3C40CCn, amount: 1000000n }
+	]
+});
+```
+
+NEM:
+
+```javascript
+const { NemFacade } = require('symbolSdk/src/facade/NemFacade');
+...
+const facade = new NemFacade('testnet');
+
+const transaction = facade.transactionFactory.create({
+	type: 'transfer_transaction_v1',
+	signerPublicKey: 'A59277D56E9F4FA46854F5EFAAA253B09F8AE69A473565E01FD9E6A738E4AB74',
+	fee: 0x186A0n,
+	timestamp: 191205516,
+	deadline: 191291916,
+	recipientAddress: 'TALICE5VF6J5FYMTCB7A3QG6OIRDRUXDWJGFVXNW',
+	amount: 5100000n
+});
+```
+
+2. Signature should always be attached via transaction factory, sample:
+```javascript
+const privateKey = new PrivateKey('EDB671EB741BD676969D8A035271D1EE5E75DF33278083D877F23615EB839FEC');
+const signature = facade.signTransaction(new NemFacade.KeyPair(privateKey), transaction);
+
+const jsonPayload = facade.transactionFactory.attachSignature(transaction, signature);

```

Comment:
It seems we need to add a `constructor` to call `attachSignature` method.

Reply:
oh, this assumes facade is already existing (like above `const facade = new SymbolFacade('testnet');`) - or were you referring to something else?

Reply:
yes, @AnthonyLaw is correct. it should be `facade.transactionFactory.constructor.attachSignature`

Reply:
I think because the `attachSignature` is a static method.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static#calling_static_members_from_a_class_constructor_and_other_methods